thirdparty/tesseract/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ ep_get_source_dir(SOURCE_DIR)
 if ("$ENV{ANDROID}" STREQUAL "1")
     set(PATCH_CMD sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/tesseract-android.patch")
```

Comment:
You need to move `COMMAND` into PATCH_CMD. Otherwise, cmake will complain on argument not found for `COMMAND`.